shacl-compact-syntax/scope.md

```diff
@@ -18,34 +18,82 @@ Note that this should be done *after* these features have been supported by the
+- [Support `xone` and more `or` cases](https://github.com/w3c/shacl/issues/12)
+- Opt out of the production of the triple `?baseUri rdf:type owl:Ontology`
+- Non-validating characteristics (e.g., generating `sh:order` triples by declaring `@order` at the top of a file/shape, allow grouping by `@group` at the top of a set of properties, and allowing a `@description` above properties).
+- [Graphical Operators](https://rawgit2.com/VladimirAlexiev/shacl/shaclc-grammars/shacl-compact-syntax/grammar/shaclc-XText.html#OP_XONE) (e.g., `><`, `⊻`, and/or `⩒`) in addition to keywords (e.g., `xone` and/or `xor`).
+  This is a way to make SHACL-C even more compact, but many people cannot type them easily. The use of well-known operators can help make code more readable, and we expect that, like all models, they are far more often read then written, but where the boundary of obscurity ends is open to discussion.
+
+### Extra SHACL Features
+
+The following features would extend SHACL, so are subject of discussion (https://github.com/w3c/shacl/issues/74).
+
+The first one comes from RDF4J.
+The rest come from [SPARQL Inferencing Notation (SPIN)](https://spinrdf.org/) (the predecessor of SHACL)
+and/or the [Allotrope](https://www.allotrope.org/) Shapes Editor (see [SHACLC.xtext](https://gitlab.com/allotrope-open-source/shape-editor/-/blob/master/src/com.osthus.shapes.shaclc.parent/com.osthus.shapes.shaclc/src/com/osthus/shapes/shaclc/SHACLC.xtext)).
+Some or all of these additions may be ignored in favour of keeping SHACL-C a simple syntax to implement and use.
+
+- Target shapes, see [rsx:targetShape](https://rdf4j.org/shacl/extensions.html) (implemented in RDF4J ShaclSail)
+- [Shape Libraries](https://rawgit2.com/VladimirAlexiev/shacl/shaclc-grammars/shacl-compact-syntax/grammar/shaclc-XText.html#ShaclDoc) (versioned). Packaging shapes into libraries enables modularized development of shapes. 
+  It is used together with [Import](https://rawgit2.com/VladimirAlexiev/shacl/shaclc-grammars/shacl-compact-syntax/grammar/shaclc-XText.html#ImportsDecl).
+  A shape library is very similar to an OWL ontology and technically works in the same way. 
+  In fact, we now use OWL ontologies because of inference from `owl:import`, but there was quite some discussion about whether the assembly of shapes is an ontology in the OWL sense. 
+  For any large scale application of SHACL, it is necessary to modularize, and it felt like this was a serious gap in the specification.
+- [Imports](https://rawgit2.com/VladimirAlexiev/shacl/shaclc-grammars/shacl-compact-syntax/grammar/shaclc-XText.html#ImportsDecl) (how is this different from `owl:imports`?)
```

Comment:
Looking at the [example](https://w3c.github.io/shacl/shacl-compact-syntax/#example), this is exactly translated to `owl:imports`. Nothing more...